### PR TITLE
4 controllers

### PR DESCRIPTION
**Pull request by OscarFMdev**
# PR description: 📄 

In this PR I:
- Added controllers for users and posts. 🎮 
- Added views and actions for index and show. 🪟 
- Nested the post's resources. 🪆 

### Thank you for reviewing my code! 🙌🏽

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
+  resources :users
+
+  root "users#index"
+
+  resources :users do
+    resources :posts
+  end
   # Defines the root path route ("/")
   # root "articles#index"
 end
```
Comment (tresorsawasawa) [inline config/routes.rb]: - [ ] As required for the project, kindly make sure `users` and `posts` routes added to your `config/routes.rb` maintained a RESTful URL corresponding to the listed **[wireframes from your project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project.md)**, i.e you do not expose unnnecessary endpoints only the 4 required URLs should be there(may be the exra `route` can be for the `root` one as you did). Once you have edited you routes configuration, you can run `rails routes | grep user` in your terminal and it should yield URLs that correspond to [these wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project.md):
You can refer this **[article](https://guides.rubyonrails.org/routing.html#configuring-the-rails-router)** to get an example on how to do that

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @OscarFMdev 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (OscarFMdev) [reply config/routes.rb]: I have fixed this issue, thank you @tresorsawasawa! 🙌🏽

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @OscarFMdev 👋, 

Well done for implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**